Say Their Names/Source/View/Home/Person/PersonPhotoCell.swift

```diff
@@ -35,10 +36,15 @@ final class PersonPhotoCell: UICollectionViewCell {
     private func setup() {
         addSubview(personImageView)
         personImageView.fillSuperview()
+        personImageView.clipsToBounds = true
     }
     
-    func setImage(_ image: UIImage?) {
-        personImageView.image = image
+    func setImage(withUrlString url: String) {
+      personImageView.kf.indicatorType = .activity
+      personImageView.kf.setImage(
+        with: URL(string: url),
+        placeholder: UIImage(named: "image-placeholder")
+      )
```

Comment:
I see this could be made into a reusable function that takes image url and placeholder image as parameters - to reduce duplicated code

Reply:
Agree, I just thought I was overthinking it at the time, that's why I left it as it is, but I'll make those updates